jugs-map.js

```diff
@@ -0,0 +1,103 @@
+        var bicycleRental = {
```

Comment:
My idea is to populate this kind of values from the actual `.MD` we have right now and therefore render the map with the JUGs